models/repo.go

```diff
@@ -198,7 +200,8 @@ func CheckRepoStats(ctx context.Context) error {
 		},
 		// Issue.NumComments
 		{
-			statsQuery("SELECT `issue`.id FROM `issue` WHERE `issue`.num_comments!=(SELECT COUNT(*) FROM `comment` WHERE issue_id=`issue`.id AND type=0)"),
+			statsQuery(fmt.Sprintf("SELECT `issue`.id FROM `issue` WHERE `issue`.num_comments!=(SELECT COUNT(*) FROM `comment` WHERE issue_id=`issue`.id AND (type=%d OR type=%d))",
+				issues_model.ConversationCountedCommentType()...)),
```

Comment:
```func statsQuery(args ...any)```

Reply:
Yes, if it's `statsQuery(string, ...any)`, then I can remove the format. But since it's `statsQuery(...any)`. It's also not perfect to create a temporary slice.

Reply:
But we should never used `ExecSQL(Sprintf(...))` (in case SQL-injection), we should always use statement arguments, right?

Reply:
ee4bf009eb